code/__DEFINES/misc.dm

```diff
@@ -434,3 +434,7 @@ GLOBAL_LIST_INIT(ghost_others_options, list(GHOST_OTHERS_SIMPLE, GHOST_OTHERS_DE
 
 //for obj explosion block calculation
 #define EXPLOSION_BLOCK_PROC -1
+
+#define BEAT_FAST 1
+#define BEAT_SLOW 2
+#define BEAT_NONE 0
```

Comment:
up to you, but inserting a comment to describe what they're for is probably a good idea